config/locales/models/implementing_organisation.en.yml

```diff
@@ -8,11 +8,6 @@ en:
         button: Remove
         success: Implementing organisation successfully removed
 
-  form:
```

Comment:
This is the only one that's making me think twice - I wonder if we should move all of the translations in this file to the `activities` translations file if we're classifying Implementing orgs as part of the activity journey?